cost/google/cud_report/README.md

```diff
@@ -0,0 +1,32 @@
+## Google Committed Use Discount (CUD) Report
+ 
+### What it does
+This policy identifies all CUDs that exist in a given GCP project and provides a report listing them all. It can optionally report on all CUDs, or only those that are active or expired.
+ 
+### Functional Details
+ 
+Uses the GCP API to get a list of all CUDs and report on them.
```

Comment:
I would add some detail here about using a service account and what perms are required for that.

Reply:
Updated file with information on creating a Service account.